Framework/qcTaskDplConfig.json

```diff
@@ -47,17 +47,8 @@
       }
     },
     "checkers_config": {
-      "numberCheckers": "1",
-      "numberTasks": "1",
-      "tasksAddresses": "tcp://localhost:5556,tcp://localhost:5557,tcp://localhost:5558,tcp://localhost:5559",
-      "checker_0": {
-        "broadcast": "0",
-        "broadcastAddress": "tcp://*:5600",
-        "id": "0"
-      },
-      "Checks": {
-        "checkMeanIsAbove/threshold": "1"
```

Comment:
Right. Though, as I look at it now, it may cause some problems. Imagine 2 tasks, that use some common check, but with different parameters. In this case, checker configuration is global, so it is not possible to do that. We need to rethink checks configuration anyway.

Reply:
agreed, actually if you look at the document I had written about checker configuration this is described there. 
```"skeletonTaskDefinition": {
    "className": "o2::quality_control_modules::daq::DaqTask",
    "moduleName": "QcDaq",
    "maxNumberCycles": "-1",
    "cycleDurationSeconds": "10",
    "objects_checks": [
      {
           "name": "fake1",
           "object": "object1",
           "moduleName": "QcExample",
           "className": "o2::quality_control_modules::example::FakeCheck",
           "parameters": {
          "xyz": "10",
          "asdf": "asdf"
           }
      },
```